state/open.go

```diff
@@ -360,6 +369,28 @@ func (st *State) modelSetupOps(args ModelArgs, controllerInheritedConfig map[str
 	return ops, nil
 }
 
+func (st *State) createDefaultStoragePoolsOps(registry storage.ProviderRegistry) ([]txn.Op, error) {
+	m := poolmanager.MemSettings{make(map[string]map[string]interface{})}
```

Comment:
You can't assign
    map[string]attrValues
to
   map[string]map[string]interface{}
(the latter being the type used in poolmanager.MemSettings)